_apidocs/regulationsgov/openapi.yaml

```diff
@@ -0,0 +1,521 @@
+#------------------
+# Temporary Notes:
+# nullable needs to be revisted.
+# All date formats are part of description. I think we can remove that.
+# TODO:  verify descriptions for attachments and display properties
+#--------------------
+openapi: 3.0.0
+info:
+  title: Regulations.gov API
+  description: >-
+    Public API for Regulations.gov
+  version: "4.0"
+servers: #do we need test endpoints mentioned here? Doesnt federalRegister uses test endpoint today?
```

Comment:
Let's remove this comment, if you want to revisit this for the docs, let's create a ticket please.